modules/cert-manager-install-console.adoc

```diff
@@ -36,6 +36,11 @@ See supported {cert-manager-operator} versions in the following "Additional reso
 +
 If the `cert-manager-operator` namespace does not exist, it is created for you.
 
+[NOTE]
+====
+During the installation, the {product-title}  web console will allow you to select between `AllNamespaces` and `SingleNamespace` installation modes. For installations with {cert-manager-operator} version 1.15.0 or later, it is recommended to choose the `AllNamespaces` installation mode. `SingleNamespace` and `OwnNamespace` support will remain for earlier versions but will be deprecated in future versions.
```

Comment:
Just checking - here it says that support for these install modes will be deprecated in future versions, while in the CLI procedure it says that support will be removed. Unless these different phrasings are intentional, I would just make the two consistent.